### ISSUE TITLE
Unable to see the selected item of select.input-sm

### ISSUE THREAD

**Issue by adam-scania** [scania/corporate-ui#450]
**Version of Corporate-ui**<br/>
3.5.0

**Describe the bug**<br/>
I'm unable to see what has been selected in small selects.

**To Reproduce**<br/>
```
<html>
    <head>
        <script src="https://static.scania.com/build/global/3.5.0/js/corporate-ui.js"></script>
    </head>
    <body class="app">
        <div class="row">
            <div class="col-md-4">
                <div class="form-group">
                    <label class="control-label">Small select</label>
                    <select class="form-control input-sm">
                        <option selected>Lorem ipsum</option>
                    </select>
                </div>
            </div>
            <div class="col-md-8">
                <table class="table-responsive">
                    <table class="table table-panel table-striped">
                        <thead>
                            <tr>
                                <th>Small select in <code>thead > tr > td</code></th>
                                <th>Small select in <code>thead > tr > th</code></th>
                            </tr>
                            <tr>
                                <td>
                                    <select class="form-control input-sm">
                                        <option selected>Lorem ipsum</option>
                                    </select>
                                </td>
                                <th>
                                    <select class="form-control input-sm">
                                        <option selected>Lorem ipsum</option>
                                    </select>
                                </th>
                            </tr>
                        </thead>
                    </thead>
                </div>
            </div>
        </div>
    </body>
</html>
```

**Expected behavior**<br/>
I expect to see the selected item.

**Screenshots**<br/>
![small-select](https://user-images.githubusercontent.com/18661799/51738178-cab7d180-208e-11e9-9a35-50c3d3a07617.png)

**Desktop (please complete the following information):**<br/>
 - OS: Windows 7
 - Browser: Chrome and Firefox
 - Version:

**Comment by scania-bootstrap** [comment]
Fixed here: https://github.com/scania/corporate-ui/pull/457

**Comment by adam-scania** [comment]
This is also an issue in panels in CUI 3.5.2.

![select-panel-heading](https://user-images.githubusercontent.com/18661799/52400446-70236a00-2abf-11e9-8449-eba7ea0700fa.png)

```
<html>
    <head>
        <script src="https://static.scania.com/build/global/3.5.2/js/corporate-ui.js"></script>
    </head>
    <body class="app">
        <div class="panel panel-primary">
            <div class="panel-heading">
                <select class="form-control input-sm">
                    <option selected>Test</option>
                </select>
            </div>
        </div>
    </body>
</html>
```

**Comment by yuyo2** [comment]
Fixed in #457
